src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreator.java

```diff
@@ -49,23 +49,22 @@ private void createBusEquation(LfBus bus,
             p.setActive(false);
         }
 
-        createGeneratorControlEquations(bus, equationSystem);
+        // maybe to fix later, but there is so part of OLF (like sensitivity) that needs a voltage target equation
```

Comment:
This is not equivalent as what we do on main branch. I understand that here you create an inactive equation for all buses. You prefer like this? No performance issue?

Reply:
In main we add an if condition on `(!equationSystem.hasEquation(bus.getNum(), AcEquationType.BUS_TARGET_V))`.

Reply:
Look at the code, this is already what is done. Creating all V equation at beginning or creating necessary equations and adding missing one at is end is exactly the same.

Reply:
yes, your are right! All bus_v equations are activated at the end.